datacube_stats/models.py

```diff
@@ -1,6 +1,10 @@
 from __future__ import absolute_import
-from datacube.model import DatasetType
+try:
```

Comment:
Curious... are there plans to rename `DatasetType` to `Product`?

Reply:
I would really really like to. The two are synonymous, and used in different places throughout the code, which makes things intensely confusing.

While I'm completely sure about the name *product*, I'm very sure that *DatasetType* is confusing.